src/ctapipe/calib/camera/tests/test_calibrator.py

```diff
@@ -130,6 +131,23 @@ def test_check_dl0_empty(example_event, example_subarray):
     assert (event.dl1.tel[tel_id].image == 2).all()
 
 
+def test_dl1_variance_calib(example_event, example_subarray):
+    # test the calibration of variance images
+    tel_id = list(example_event.r0.tel)[0]
+    calibrator = CameraCalibrator(
+        subarray=example_subarray,
+        image_extractor=VarianceExtractor(subarray=example_subarray),
+        apply_waveform_time_shift=False,
+    )
+    calibrator(example_event)
+    image = example_event.dl1.tel[tel_id].image
+    assert image is not None
+    assert image.shape == (
```

Comment:
Add a test with the LST (two gains)

Reply:
ok

Reply:
would it make sense to add a test for other extractor with the LST?

Reply:
You could use a parametrized test here to test all telescope types.  (`@pytest.mark.parametrized`). See for example the code in `ctapipe/instrument/tests/test_telescope.py`